website/views.py

```diff
@@ -88,6 +94,70 @@ def post(self, request):
                     return redirect('website:index')
         return render(request, self.template_name, { 'form' : form })
 
+class ProjectListView(EmailRequiredMixin, ListView):
```

Comment:
the new mixin also needs to be added to the project selection creation page - which is currently a function and not a class (it needs converted)

Reply:
Another option is to make a [decorator](https://github.com/django/django/blob/master/django/contrib/auth/decorators.py) and decorate that function instead of converting it